### ISSUE TITLE
decorator.oauth_required should accept and pass kwargs to decorated method

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#110]
```
I realize that the decorator was written specifically with 
webapp.RequestHandler in mind, but GAE includes webapp2 which allows request 
handlers to have kwargs that derive from routing rules, it would be nice if the 
decorator would be kind to kwargs passed in from the above application down to 
the actual handler.
```

Original issue reported on code.google.com by `rtp@google.com` on 1 Mar 2012 at 12:16

**Comment by GoogleCodeExporter** [comment]
```
http://codereview.appspot.com/5712049/
```

Original comment by `jcgregorio@google.com` on 1 Mar 2012 at 3:40
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Woo! Thanks!
```

Original comment by `rtp@google.com` on 1 Mar 2012 at 4:12

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision cb1628802b10.
```

Original comment by `jcgregorio@google.com` on 1 Mar 2012 at 4:12
- Changed state: **Fixed**
